Terminal.Gui/Views/Menu/MenuBar.cs

```diff
@@ -559,7 +559,7 @@ private void CloseOtherOpenedMenuBar ()
         if (Application.Top is { })
         {
             // Close others menu bar opened
-            Menu? menu = Application.Top.Subviews.FirstOrDefault (v => v is Menu m && m.Host != this && m.Host.IsMenuOpen) as Menu;
+            Menu? menu = Application.Top.SubViews.FirstOrDefault (v => v is Menu m && m.Host != this && m.Host.IsMenuOpen) as Menu;
             menu?.Host.CleanUp ();
```

Comment:
We have Adornment.Parent and Menu.Host and other similar.

We should standardize on only 2 conditions

- View that is added like normal to the hierarchy, use `SuperView`
- View that is special e.g. a border, a menu, a context popup etc etc, use standard term e.g. `Parent`

Even better if we add an interface for these views `IViewSubcomponent` or something

Reply:
I agree. The menu/popup menu is being fixed in

- #2975